opal/mca/common/verbs/common_verbs_port.c

```diff
@@ -68,6 +68,14 @@ int opal_common_verbs_port_bw(struct ibv_port_attr *port_attr,
         /* EDR: 25.78125 Gbps * 64/66, in megabits */
         *bandwidth = 25000;
         break;
+    case 64:
+        /* HDR: 50Gbps * 64/66, in megabits */
+        *bandwidth = 50000;
+        break;
+    case 128:
```

Comment:
Two questions:

1. According to https://github.com/torvalds/linux/blob/master/include/rdma/ib_verbs.h#L428-L435, I do not see values for 64 and 128 defined in the kernel ABI for the `active_speed` field when querying IB port attributes.  Where are you getting these 64/128 values from?

2. As a secondary (but related) issue, is there a better way than using hard-coded integer values here in `common_verbs_port.c`?  On the kernel side, we have the nice `IB_SPEED_*` enums; do public equivalents exist in userspace that we can use here in Open MPI?  (ditto for `IB_WIDTH_*`)

Reply:
Just to note that we have been pointed to this PR by Mellanox about OMPI not working with CX5 cards and ROCE.

However, we do not see the backtrace that @dsharma283 reports, just the error:

`--------------------------------------------------------------------------
WARNING: There was an error initializing an OpenFabrics device.

  Local host:   spartan-gpgpu008
  Local device: mlx5_0
--------------------------------------------------------------------------
--------------------------------------------------------------------------
WARNING: There was an error initializing an OpenFabrics device.

  Local host:   spartan-gpgpu007
  Local device: mlx5_0
--------------------------------------------------------------------------
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
srun: error: spartan-gpgpu008: task 1: Exited with exit code 17
srun: error: spartan-gpgpu007: task 0: Killed`